test/unit/sql/rounds.js

```diff
@@ -433,6 +224,117 @@ describe('Rounds-related SQL triggers', function () {
 			})
 		});
 
+		it('modules.delegates.__private.delegatesList should be equal with one generated with Lisk-Core 0.9.3', function () {
```

Comment:
When we are describing a property or function, I think this should be in the form as below:

```js
describe('modules.delegates.__private.delegatesList', function () {

  // Sourced from lisk-core 0.9.3
  var expectedDelegatesOrder = []; // Possibly loaded as a JSON fixture
  
  it('should be an array');

  it('should have a length of 101');

  it('should consist of public keys');

  it('should be ordered as expected');
});
```